revolv/static/nav.scss

```diff
@@ -37,6 +37,12 @@ nav.header {
     .user-controls > .dropdown a.js-generated {
         display: none;
     }
+
+    /* Overriding Foundation colors */
```

Comment:
This seems a bit much: I guess we probably need all these selectors due to CSS specificity right? Can we at least have a comment explaining what this is doing specifically/which element it's actually styling?